### ISSUE TITLE
Visualise network traffic flows

### ISSUE THREAD

**Issue by dms1981** [ministryofjustice/modernisation-platform#3116]
## User Story

As a Modernisation Platform customer 
I want to visualise my traffic flows through a centralised dashboard
So that I can understand how my resources are consuming network bandwidth

## User Type(s)

Modernisation Platform Customer
Modernisation Platform Engineer

## Value

On a recent call around the CCMS-EBS migration, questions were raised around how instances were consuming bandwidth. We log network traffic through AWS VPC Flow Logs but do not have an easy way of interrogating them to retrieve aggregated figures; our use case has historically been to confirm if flows have been successful, or if they have been blocked by security groups.

It is possible to review these flow logs to give us information on bandwidth consumed by an instance in a certain time window, or even to show us how much bandwidth is in use to or from certain endpoints. AWS have an example architecture published [here](https://aws.amazon.com/blogs/big-data/analyze-and-visualize-your-vpc-network-traffic-using-amazon-kinesis-and-amazon-athena/) which could be adapted by us into Terraform, or another option that uses [AWS OpenSearch](https://aws.amazon.com/blogs/big-data/stream-vpc-flow-logs-to-amazon-opensearch-service-via-amazon-kinesis-data-firehose/).

This would then allow us, or potentially our customers, to review flow log information in a more granular fashion than a simple CloudWatch metric like `NetworkIn` or `NetworkOut`.

### Hypothesis 
If we give our customers the ability to view flow log information
Then they will be able to resolve issues with less MP Engineer involvement

## Definition of done

- [ ] relevant solution decided upon
- [ ] solution implemented in core-logging account
- [ ] tests are green

## Reference

[How to write good user stories](https://www.gov.uk/service-manual/agile-delivery/writing-user-stories)

[https://aws.amazon.com/blogs/big-data/analyze-and-visualize-your-vpc-network-traffic-using-amazon-kinesis-and-amazon-athena/](https://aws.amazon.com/blogs/big-data/analyze-and-visualize-your-vpc-network-traffic-using-amazon-kinesis-and-amazon-athena/)

[https://aws.amazon.com/blogs/big-data/stream-vpc-flow-logs-to-amazon-opensearch-service-via-amazon-kinesis-data-firehose/](https://aws.amazon.com/blogs/big-data/stream-vpc-flow-logs-to-amazon-opensearch-service-via-amazon-kinesis-data-firehose/)

**Comment by SimonPPledger** [comment]
would be good but a lower priority

**Comment by dms1981** [comment]
In really brief terms, it would be great if we could use Grafana/Observability Platform to allow customers to plug in source and destination network addresses to see if there's traffic passing successfully between hosts. This issue would find a way for us to deliver this.
